packages/react-components/src/components/DomainPermissions.tsx

```diff
@@ -84,13 +84,13 @@ const DomainPermissionsContainer = (props: DomainPermissionsProps): JSX.Element
         </Stack>
       </Stack>
       <Stack styles={textContainerStyles}>
-        <Text styles={primaryTextStyles}>{_formatString(strings.primaryText, { appName: appName })}</Text>
-        <Text styles={secondaryTextStyles}>{strings.secondaryText}</Text>
+        <Text styles={primaryTextStyles}>{_formatString(strings?.primaryText ?? '', { appName: appName })}</Text>
```

Comment:
odd situation here but what happens if the strings object is undefined here? does it just display the appName? I think `_formatString` tries to put those two strings together by the `appName` attribute in the primaryText string. does it just return an empty string in this case?

Reply:
in the code below string is populated by locale so it wont be undefined, just doing this here to get pass lint errors. We need string prop to be optional

Reply:
fixed